packages/sdk/build.js

```diff
@@ -0,0 +1,86 @@
+// this file is adapted from https://blog.unterholzer.dev/cross-compatible-typescript-libraries/
```

Comment:
Out of curiousity, why is this necessary -- shouldn't rollup create two separate builds for us and we should just need to point module and main at the different ones?

Reply:
This was copied from a tutorial. The problem is the sdk is using tsdx, which outputs the build to a single folder and will override the results of a previous build.  So this hacks around this by moving files around so previous build does not get overrwritten. There may be a better way to do this though

Reply:
oh this file also copies the types file so that both builds have the same interface